HLTrigger/Tools/bin/hltTimingSummary.cpp

```diff
@@ -1880,16 +1880,16 @@ int main(int argc, char ** argv) {
       }
     }
     
-    char value[10] ; 
+    char value[330] ; 
```

Comment:
I'm not sure I agree... anyway, why is this an error and not a warning ? `snprintf` should just truncate the output, right ?